src/frontends/tensorflow/src/op/slice.cpp

```diff
@@ -18,11 +18,22 @@ OutputVector translate_slice_op(const NodeContext& node) {
     auto start = node.get_input(1);
     auto size = node.get_input(2);
 
-    auto stop = make_shared<Add>(start, size);
+    // compute stop values in case non-negative sizes
+    auto stop_pos = make_shared<Add>(start, size);
 
-    auto one = make_shared<Constant>(element::i64, Shape{1}, 1);
-    auto shape = make_shared<ShapeOf>(start);
-    auto step = make_shared<Broadcast>(one, shape);
+    // compute stop values in case positive sizes
+    auto input_shape = make_shared<ShapeOf>(input);
+    auto one = make_shared<Constant>(size.get_element_type(), Shape{}, 1);
+    auto stop_neg = make_shared<Add>(make_shared<Add>(make_shared<ConvertLike>(input_shape, size), size), one);
```

Comment:
it looks like tf.slice doesn't support negative sizes
![image](https://user-images.githubusercontent.com/21181928/183413431-bd3834cd-5a9f-4603-9f9d-4339a0a9c2e1.png)

Reply:
except -1 :)

Reply:
it looks like the only one negative value is supported (-1)
can we simplify this?

Reply:
Simplified. Fixed.